unified/install.sh

```diff
@@ -108,7 +108,8 @@ if ! $skip_systemd_check && [ ! -d /run/systemd/system/ ]; then
 fi
 
 if ! scylla-jmx/select-java -version > /dev/null; then
-    echo "Please install openjdk-8 or openjdk-11 before running install.sh."
+    echo "Please ensure you use either java-1.8.0-openjdk or java-11-openjdk before running install.sh:"
+    echo "check if you have this version installed and if it's set as a default tool (JAVA_HOME env var + javac version)"
```

Comment:
@ptrsmrn hi Piotr, the `scylla-jmx/select-java` script should be able to find java-11 or java-8 is either of them is installed. see https://github.com/scylladb/scylla-jmx/blob/53696b133284363cddf689ebfb1cf695d18c0fe4/scripts/select-java#L7-L16 . so this change does not apply to the master branch or the branches which includes the change in its jmx submodule, i think.

Reply:
OK, thanks for pointing that out. I'll close this PR then.